### ISSUE TITLE
About pose tracking

### ISSUE THREAD

**Issue by CheungBH** [open-mmlab/mmpose#510]
Hello. After running the training steps with the posetrack18 dataset and resnet50, I found that there is a "pred" folder that contains the results files of the testing dataset.
I noticed that the id of the humans is also assigned, and I wonder which algorithm is used to generate the tracked id? Are there any references? 
I am looking forward to your reply!

**Comment by jin-s13** [comment]
For now, the human ids are assigned just at random. We do not support evaluating the "tracking" part of posetrack18, and only image-based pose estimation results are evaluated. 

If you would like to evaluate the pose tracking results, please follow the [posetrack demo](https://github.com/open-mmlab/mmpose/blob/master/demo/2d_pose_tracking_demo.md) and generate the pose tracking results. Good luck!

**Comment by CheungBH** [comment]
Thank you for your reply! 
BTW, I found that the "pred" folder contains the results of VALIDATION data instead of test data. But when I change the anno file to test data, an error occurs.
![image](https://user-images.githubusercontent.com/39186017/109746557-881f3680-7c10-11eb-945c-e32371ad87fd.png)
I wonder what to do to generate the JSON files containing the results of testing data?

**Comment by jin-s13** [comment]
FileNotFoundError.  You have to generate the `posetrack18_test.json` by yourself.

**Comment by CheungBH** [comment]
But the annotation of the test set is not provided. Actually, I just want to generate the keypoint results and sent them to the evaluation server. I don't want to compare them with the ground truth. Is it feasible?

**Comment by jin-s13** [comment]
Yes. It is possible. But some minor modifications are needed. 

We do not need gt annotations to run inference. But we need human detected bounding boxes.

We have supported evaluating on COCO test-dev, by uploading generated results to the server. Please refer to this [issue] (https://github.com/open-mmlab/mmpose/issues/410)and generate required files.

**Comment by CheungBH** [comment]
Thanks for your reply! 
BTW, I am want to upload the results to the evaluation server, but I found that the evaluation server of posetrack has been closed. I wonder whether you have uploaded and evaluated the results currently?

**Comment by jin-s13** [comment]
No, we have not uploaded the results to the server.

**Comment by WEIZHIHONG720** [comment]
the track_ids  are assigned jst at random, does this have an effect on the results?
